src/classBased/Components/InputTodo.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable */
+import React, { Component } from 'react';
+
```

Comment:
Please try not to disable linters, it would be better if you fix linter errors instead